bolt/lib/Core/DIEBuilder.cpp

```diff
@@ -496,9 +506,6 @@ uint32_t DIEBuilder::finalizeDIEs(
   }
 
   Die.setSize(CurSize);
-  if (NameEntry)
```

Comment:
why is this code removed?

Reply:
I no loner use explicit vector and it is handled fully by recursion.

Reply:
SG